Cargo.lock

```diff
@@ -805,6 +897,16 @@ dependencies = [
  "hashbrown 0.11.2",
 ]
 
+[[package]]
+name = "borsh"
+version = "0.10.3"
```

Comment:
Please investigate upgrading this dependency.

Reply:
This has been [done on the Aurora side](https://github.com/aurora-is-near/aurora-engine/pull/907). But it hasn't been released as a tag yet on GitHub. This will be resolved when we next upgrade the Aurora dependency.